src/App.js

```diff
@@ -117,6 +119,7 @@ export default class App extends React.Component<{}, AppState> {
 
     handleClickConnectDash = () => {
         this.setState({
+            modalIsShowing: false,
```

Comment:
Move this `modalIsShowing: false` line down 2 lines, to below `showDashConnectionError: false`. To group them together.